### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by lucalinux**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,3 +5,5 @@ public-java-material-lucalinux created by GitHub Classroom
 See new material and code
 
 Added material
+
+modification from Linux Debian
```
Comment (marioluca) [inline README.md:9]: Violates policies.

File: README.md [hunk 2]
```diff
@@ -5,3 +5,5 @@ public-java-material-lucalinux created by GitHub Classroom
 See new material and code
 
 Added material
+
```
Comment (marioluca) [inline README.md:8]: Check this approach. I don't think it is goint to work.

Comment (lucalinux) [reply README.md:8]: Fixed.

File: README.md (hunk 1 above)
Comment (lucalinux) [reply README.md:9]: Fixed.

**Pull request closed without merging**